cmd/publish.go

```diff
@@ -37,6 +40,7 @@ func setupPublishCommand() *cobra.Command {
 func publishCommandAction(cmd *cobra.Command, args []string) error {
 	cmd.Println("Publish the package")
 
+	fork, _ := cmd.Flags().GetBool(cobraext.ForkFlagName)
```

Comment:
Any reason not to check for error here and fail using `cobraext.FlagParsingError`?